### PR TITLE
Add steps to use Tempo as tracing storage via Tempo operator

### PR DESCRIPTION
**Pull request by iblancasa**
Add a new section to explain how to use Tempo (with the Tempo Operator) as tracing storage.

Later, we can add more sections about how to do the same using Heml or other deployment modes that can be interesting for the users.

### CODE REVIEW

**Comment by jmazzitelli** [comment]
netlify link for reviewers to look at: https://deploy-preview-677--kiali.netlify.app/docs/configuration/p8s-jaeger-grafana/jaeger/#use-jaeger-frontend-with-grafana-tempo-tracing-backend

File: content/en/docs/Configuration/p8s-jaeger-grafana/jaeger.md [hunk 1]
```diff
@@ -52,3 +52,64 @@ If your Jaeger setup differs significantly from the sample add-ons, make sure
...
+Since Tempo is not yet part of the built-in addons that are part of Istio, you
+need to manage your Tempo instance.
+
+### Tempo Operator
+
+The [Tempo Operator for Kubernetes](https://github.com/grafana/tempo-operator)
+provides a native Kubernetes solution to deploy Tempo easily in your system.
+
+After installing the Tempo Operator in your cluster, you can create a new
+Tempo instance with the following CRD:
```
Comment (jmazzitelli) [inline content/en/docs/Configuration/p8s-jaeger-grafana/jaeger.md]: This is not a CRD (a CustomResourceDefinition).. this is an instance of the tempo CRD - aka a "Custom Resource" or CR:

```suggestion
Tempo instance with the following CR:
```

**Review by jmazzitelli — CHANGES_REQUESTED** [review]
I haven't actually ran any of this, but other than the trivial change request, the documentation itself looks good.

**Comment by jshaughn** [comment]
Thanks @iblancasa !  Last week we had a user get things working as [described here](https://github.com/kiali/kiali/issues/5849#issuecomment-1624311897), does that look correct to you, and does the PR cover his configuration?

File: content/en/docs/Configuration/p8s-jaeger-grafana/jaeger.md (hunk 1 above)
Comment (iblancasa) [reply content/en/docs/Configuration/p8s-jaeger-grafana/jaeger.md]: Ooops. I typed too fast. Thanks!

**Comment by iblancasa** [comment]
> Thanks @iblancasa ! Last week we had a user get things working as [described here](https://github.com/kiali/kiali/issues/5849#issuecomment-1624311897), does that look correct to you, and does the PR cover his configuration?

I covered here the creation of a Tempo instance using the Kubernetes Operator. That description uses Helm. I think we should cover both. Also [the method described in the official Tempo documentation](https://github.com/kiali/kiali/issues/5848#issuecomment-1518028505). But as part of different PRs to make it easier to give feedback/review.

Sorry anyway for the delay. Lot of stuff in progress.

**Comment by jshaughn** [comment]
> I covered here the creation of a Tempo instance using the Kubernetes Operator. That description uses Helm. I think we should cover both. Also https://github.com/kiali/kiali/issues/5848#issuecomment-1518028505. But as part of different PRs to make it easier to give feedback/review.

Ah, I see, thanks.  This looks good to ne but I added @aljesusg as a reviewer.  GIven your feedback I added two more subtasks to the Epic https://github.com/kiali/kiali/issues/5850.  One for Helm and one for Tempo docs.  Will you be able to help with those as well?

**Review by jmazzitelli — APPROVED** [review]
looks good!

**Review by aljesusg — APPROVED** [review]
LGFM

**Comment by iblancasa** [comment]
> Ah, I see, thanks. This looks good to ne but I added @aljesusg as a reviewer. GIven your feedback I added two more subtasks to the Epic [kiali/kiali#5850](https://github.com/kiali/kiali/issues/5850). One for Helm and one for Tempo docs. Will you be able to help with those as well?

Yes. I'll try my best :D

**Pull request merged**